packages/react/react/src/hooks/useDebounce.ts

```diff
@@ -12,12 +12,16 @@ import { usePreservedCallback } from './usePreservedCallback';
  *   getV2Logger().log(schemaId, parameter);
  * }, 500);
  */
-export function useDebounce<F extends (...args: any[]) => any>(callback: F, wait: number) {
+export function useDebounce<F extends (...args: any[]) => any>(
+  callback: F,
+  wait: number,
+  options?: Parameters<typeof debounce>[2]
```

Comment:
My recommendation is using [default function parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters).
```typescript
function useDebounce<F extends (...args: any[]) => any>(
  callback: F,
  wait: number,
  options: Parameters<typeof debounce>[2] = {}
);
```

Reply:
thx! I learned from you
[fix: @toss/react useDebounce options use preservedOptions](https://github.com/toss/slash/pull/95/commits/cf56a618e3c87a3e34c166b30d26afe41f4b9548)
[cf56a61](https://github.com/toss/slash/pull/95/commits/cf56a618e3c87a3e34c166b30d26afe41f4b9548)